addons/account/models/account_bank_statement_line.py

```diff
@@ -237,12 +246,16 @@ def _compute_running_balance(self):
                     LEFT JOIN account_bank_statement st ON st.id = st_line.statement_id
                     WHERE
                         st_line.internal_index <= %s
-                        AND move.journal_id = %s
```

Comment:
it's a shame we are still joining the `account_move` table just for forwarding the state :S
The Pkey of `account_move` should normally be in buffers, so it's rarely a cold access.